python/src/LOLAVoronoi_doc.i.in

```diff
@@ -0,0 +1,161 @@
+
+The input sample Voronoi cell relative volumes are written:
+
+.. math::
+
+    v(\vect{p}_i) = \int_{\vect{x} \in \Rset^d} 1_{i = \argmin_{j \in [1, N]} ||\vect{p}_j - \vect{x}||_2} \inputMeasure(\vect{x}) dx
+
+and are approximated using Monte Carlo with a number of samples set by the :class:`~openturns.ResourceMap` entry `LOLAVoronoi-DefaultVoronoiSamplingSize`.
+
+For the nonlinearity criterion the algorithm only uses the data available and chooses a
+set of neighbouring samples :math:`N(\vect{p}_i) = {\vect{p}_{i1}, \hdots, \vect{p}_{im}}`
+with :math:`m = 2 d` to build the gradient approximation :math:`\vect{J}`.
+
+The least-squares method to compute the gradient is set by the :class:`~openturns.ResourceMap` entry `LOLAVoronoi-DecompositionMethod`.
+
+The LOLA algorithm local nonlinearity score is given by:
+
+.. math::
+
+    e(\vect{p}_r) = \sum_{i=1}^m |\model(\vect{p}_{ri}) - (\model(\vect{p}_{r}) + \vect{J} (\vect{p}_{ri} - \vect{p}_{r}))|
+
+Now the two metrics are combined into the hybrid score given by:
+
+.. math::
+
+    h(\vect{p}_i) = v(\vect{p}_i) + \frac{e(\vect{p}_i)}{\sum_{j=1}^n e(\vect{p}_j)}
+
+To generate :math:`n_{new}` samples, a number of random points are generated close to
+the :math:`n_{new}` existing points with highest hybrid score.
+The random points selected are the ones inside the Voronoi cells and the farthest
+away from :math:`\vect{p}_i` and its neighbours.
+
+Parameters
+----------
+x : 2-d sequence of float
+    Initial input sample
+y : 2-d sequence of float
+    Initial output sample
+distribution : :class:`~openturns.Distribution`
+    Distribution to generate new input samples
+
+Notes
+-----
+Various :class:`~openturns.ResourceMap` entries allow for a more fine-grained control over the algorithm:
+
+- The entry `LOLAVoronoi-DefaultNeighbourhoodCandidatesNumber` entry, see :meth:`setNeighbourhoodCandidatesNumber`.
+- The entry `LOLAVoronoi-DefaultVoronoiSamplingSize` entry, see :meth:`setVoronoiSamplingSize`.
+- The entry `LOLAVoronoi-MaximumCombinationsNumber` controls the maximum
+  number of neighbourhood combinations explored in the gradient estimation for the exploitation criterion.
+- The entry `LOLAVoronoi-UseTruncatedDistribution` controls whether the :class:`~openturns.TruncatedDistribution`
+  distribution is used to sample neighbourhoods (by CDF inversion when the copula is independent)
+  or if we want to perform rejection sampling instead.
+- The entry `LOLAVoronoi-NonLinearityAggregationMethod` controls how the non-linearity score is computed
+  across multiple outputs from the gradient vectors: either `Maximum` to distinguish the most nonlinear regions (default)
+  or `Average` that will smooth the score but can be a good alternative when there are only a few outputs.
+
+Examples
+--------
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> f1 = ot.SymbolicFunction(['a0', 'a1'], ['-4 * exp((-25 / 8) * (a0^2 + a1^2)) + 7 * exp((-125 / 4) * (a0^2 + a1^2))'])
```

Comment:
This line is too long.

![image](https://github.com/user-attachments/assets/d65a7368-2575-4ed5-a770-27e76e1292cc)